remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/ProtobufMetadataManagerInterceptor.java

```diff
@@ -315,8 +319,8 @@ public Object visitRemoveCommand(InvocationContext ctx, RemoveCommand command) t
             StringBuilder sb = new StringBuilder();
             for (FileDescriptor fd : serializationContext.getFileDescriptors().values()) {
                if (fd.isResolved()) {
-                  cmd = commandsFactory.buildRemoveCommand(fd.getName() + ERRORS_KEY_SUFFIX, null, flagsBitSet);
-                  invoker.invoke(ctx, cmd);
+                  writeCommand = commandsFactory.buildRemoveCommand(fd.getName() + ERRORS_KEY_SUFFIX, null, flagsBitSet);
```

Comment:
Actually it shouldn't be necessary to set the topology here. The command will go through the `StateTransferInterceptor`, which will both set the topology and retry if necessary, so I'll remove all the `setTopologyId()` calls from this interceptor.